src/components/dialog-full-screen/dialog-full-screen-test.stories.tsx

```diff
@@ -141,6 +143,38 @@ Nested.parameters = {
   },
 };
 
+export const WithFullWidthHeader: StoryType = {
+  render: (args) => {
+    const { children, ...rest } = args;
+    return <DialogFullScreen {...rest}>{children}</DialogFullScreen>;
+  },
+  args: {
+    children: "Content",
+    open: true,
+    title: (
+      <Box maxWidth="750px" width="100%" data-testid="test">
+        <StepFlow
+          category="category"
+          title="title"
+          currentStep={1}
+          totalSteps={6}
+          showProgressIndicator
+        />
+      </Box>
+    ),
+    showCloseIcon: false,
+    onCancel: () => {},
+  },
+  decorators: [
+    (Story) => (
+      <Box height="900px" width="100%">
+        <Story />
+      </Box>
+    ),
+  ],
+};
+WithFullWidthHeader.storyName = "with full width header";
```

Comment:
**suggestion(non-blocking)**: as the header is now full width by default, what do you think about updating the name of this story to reflect this?

```suggestion
WithStepFlowInHeader.storyName = "with StepFlow in header";
```